xds/src/main/java/io/grpc/xds/XdsClientImpl.java

```diff
@@ -67,7 +68,8 @@
 /**
  * XdsClient implementation for client side usages.
```

Comment:
Can you remove "for client side usages" from this comment? I forgot to remove it when I refactored this.

Reply:
Done